clients/src/main/resources/common/message/StreamsInitializeRequest.json

```
@@ -0,0 +1,64 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 77,
+  "type": "request",
+  "listeners": ["zkBroker", "broker"],
+  "name": "StreamsInitializeRequest",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "GroupId", "type": "string", "versions": "0+", "entityType": "groupId",
+      "about": "The group identifier." },
+    { "name":  "Topology", "type": "[]Subtopology", "versions": "0+",
+      "about": "The sub-topologies of the streams application.",
+      "fields": [
+        // We are using strings here, to avoid fixing a structure on task IDs which will
+        // allow introducing stable naming for subtopologies in the future.
+        { "name": "Subtopology", "type": "string", "versions": "0+",
+          "about": "String to uniquely the sub-topology. Deterministically generated from the topology" },
+        { "name": "Partitions", "type": "int32", "versions": "0+",
```

I would use a more meaningful name:

```suggestion
        { "name": "PartitionCount", "type": "int32", "versions": "0+",
```
or
```suggestion
        { "name": "numberOfPartitions", "type": "int32", "versions": "0+",
```